source/agora/test/Base.d

```diff
@@ -1051,14 +1051,14 @@ public class TestAPIManager
 
     ***************************************************************************/
 
-    void addBlock (bool no_txs = false,
+    public void addBlock (bool no_txs = false,
```

Comment:
Those changes, while welcome, would be better in an isolated "Style fixes" commit.